### PR TITLE
Skip "original form" and "help page url" links if the URL isn't valid

### LINKED ISSUE #86: Don't direct people to invalid URLs
If the metadata doesn't contain a valid URL, we shouldn't let people click on it.

Example: the Appeal or Stay Your Eviction help URL is "n/a"

image

Since we're automating this landing page, we should consider adding a bit of data validation.

### PR DESCRIPTION
**Pull request by nonprofittechy**
Fix #86

### CODE REVIEW

**Review by samglover — APPROVED** [review]
Looks good. The error is a Prettier formatting error that should be easy to fix.

**Comment by nonprofittechy** [comment]
Trying a `npm format` to fix the prettier error. Looks like it's overinclusive, but hopefully OK

File: src/app/[path]/forms/[form]/page.tsx [hunk 1]
```diff
@@ -1,4 +1,3 @@
-// Example: courtformsonline.org/ma/forms/[form-slug]
```
Comment (samglover) [inline src/app/[path]/forms/[form]/page.tsx:1]: I'd prefer to leave this comment in place. It helps me find my way around the various page.tsx templates. But it's not essential.

**Review by samglover — COMMENTED** [review]
Thx!

**Pull request merged**